libkbfs/md_journal.go

```diff
@@ -479,15 +490,19 @@ func (j *mdJournal) convertToBranch(
 		}
 		mdsToRemove = append(mdsToRemove, newID)
 
-		err = tempJournal.append(brmd.RevisionNumber(), newID)
+		// No need to try and preserve unknown fields from the
```

Comment:
I was thinking of fields that depend on the MdID somehow. Yeah, maybe it suffices to note that new fields shouldn't do so. I'll do that in a follow-up PR and change it to preserve.